bootloader/wscript

```diff
@@ -417,6 +422,23 @@ def set_arch_flags(ctx):
         ctx.env.WINRC_SRC_F = '-i'
 
 
+# A bare minimum zlib program. Testing zlib-dev availability requires
+# explicitly using libz defined symbols instead of just compiling
+# 'include <zlib.h>' with `-lz` because zlib's development headers may be
+# installed but with libz.so `sstrip`-ed making it not linkable to.
+ZLIB_TEST_FRAGMENT = r"""
+#include <stdio.h>
+#include <zlib.h>
+
+int main (int argc, char **argv)
+{
+	printf("Hello\n");
+	printf("Zlib: %s\n", zlibVersion());
+	return 0;
+}
+"""
+
+
```

Comment:
Let's simplify that a bit further:
```c
#include <stdio.h>
#include <zlib.h>

int main ()
{
    printf("%s\n", zlibVersion);
    return 0;
}
```

to avoid warnings-as-errors on unused arguments.

Reply:
Actually, we can probably also drop the `printf()` (and stdio include), because compiler should not optimize the function call away (even if we don't use its return value).